src/test/java/com/agiletec/plugins/jacms/aps/system/services/contentmodel/ContentModelTest.java

```diff
@@ -0,0 +1,32 @@
+package com.agiletec.plugins.jacms.aps.system.services.contentmodel;
+
+import junit.framework.TestCase;
+
+public class ContentModelTest extends TestCase {
```

Comment:
@w-caffiero-entando We must verify the case where the modelId parameter is null because in this case the system (the "Content Viewer" widget in particular) uses the default model

Reply:
ok, so <null> should be an acceptable value for isValidModelIdParam